core/pkg/util/promutil/promutil.go

```diff
@@ -0,0 +1,120 @@
+package promutil
```

Comment:
This entire utility was lifted from opencost methods in the `prom` package (which remains in opencost). 

As of now, this is duplicated code, and the opencost dependents should leverage `promutil` instead. Must make this change before merge.